server/server/models/__init__.py

```diff
@@ -2,11 +2,14 @@
 SQLAlchemy models for database
 """
 import transaction
-import zope.sqlalchemy
+
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import scoped_session, sessionmaker, configure_mappers
+from sqlalchemy import *
```

Comment:
Please never use wildcard imports because it usually is a source of errors hard to debug, and even [PEP-8](https://www.python.org/dev/peps/pep-0008/#imports) asks to do not do this:
> Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools. There is one defensible use case for a wildcard import, which is to republish an internal interface as part of a public API (for example, overwriting a pure Python implementation of an interface with the definitions from an optional accelerator module and exactly which definitions will be overwritten isn't known in advance).